app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2 class='log-title'>Log in</h2>
```

Comment:
(The previous reviewer could not review this aspect because you still needed to configure authentication parameters correctly to make your app work as intended) 
- [ ] Great job for making devise parameters work correctly. This activity requires the modification of the views in devise for the registration and login to match the styling of the [provided wireframe(s)](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md). Kindly ensure that the form (inputs and buttons) stretches across the screen and is centered accordingly. Kindly ensure to improve upon the same.

**Your login view:**
![image](https://user-images.githubusercontent.com/12745474/204063950-e7774a1e-50e6-49e6-97a6-c19aa272e1f3.png)

**Wireframe login view:**
![image](https://user-images.githubusercontent.com/12745474/192667669-04020765-a648-4c50-ab45-7d68eda203e7.png)

Reply:
Centered and stretched across the screen.